app/scripts/migrations/105.ts

```diff
@@ -1,21 +1,37 @@
 import { EthAccountType } from '@metamask/keyring-api';
-import { InternalAccount } from '@metamask/keyring-internal-api';
 import { sha256FromString } from 'ethereumjs-util';
 import { v4 as uuid } from 'uuid';
 import { cloneDeep } from 'lodash';
+import { Json } from '@metamask/utils';
 import { ETH_EOA_METHODS } from '../../../shared/constants/eth-methods';
 
 type VersionedData = {
   meta: { version: number };
   data: Record<string, unknown>;
 };
 
-type Identity = {
+export type Identity = {
   name: string;
   address: string;
   lastSelected?: number;
 };
 
+export type InternalAccount = {
+  type: string;
+  id: string;
+  options: Record<string, Json>;
+  metadata: {
+    name: string;
+    importTime: number;
+    keyring: {
+      type: string;
+    };
+    lastSelected?: number;
+  };
+  address: string;
+  methods: string[];
+};
```

Comment:
We duplicate the `InternalAccount` prior to the `scopes` addition. Also, using hardcoded types make sure that we are using the original type when that migration was first created.

Reply:
I think it can be confusing to give it the same name

Reply:
Yes, you're totally right, let me change that